docs/fdp-overview.markdown

```diff
@@ -0,0 +1,168 @@
+# Overlay Method Selection
```

Comment:
Maybe you could include a short introduction with a description of the two overlays before jumping to the explanation of how we select one of them...